Documentation/README.md

```diff
@@ -0,0 +1,19 @@
+# Documentation
```

Comment:
I'm not sure if having this README.md file is necessary, since it's entire content is duplicated in root README.md

Reply:
Yes, but I think it's still nice to have something to click on when you don't navigate from the main README but just move into the `Documentation` folder. E.g., the GIFs are linked and here.